bottle_server.py

```diff
@@ -71,6 +71,16 @@ def get_exec():
     result = stdout
   return result
 
+# the method sendMetaData returns the MetaData to the Rise4fun website
+# This metadata is found on the file MetadataInfo.json
+# If you want more information about the Metadata. 
+# This can be found on http://www.rise4fun.com/dev.
+@route('/metadata')
+def sendMetaData():
```

Comment:
changed the name of the function to rise4fun_send_metadata, and added if statement to  check if isRise4fun is true, otherwise abort